packages/yoastseo/src/languageProcessing/helpers/sanitize/removePunctuation.js

```diff
@@ -1,5 +1,11 @@
 import { unifyNonBreakingSpace } from "./unifyWhitespace";
 
+export const punctuationRegexString = "\\–\\-\\(\\)_\\[\\]’‘“”〝〞〟‟„\"'.?!:;,¿¡«»‹›\u2014\u00d7\u002b\u0026\u06d4\u061f\u060C\u061B\u3002\uff61" +
```

Comment:
export this string so I can use it elsewhere.
Question for reviewer: I am on the fence on whether it is better to put this string in a separate config file. What do you think is best.

Reply:
I created  a separate file for `packages/yoastseo/src/languageProcessing/languages/en/config/regularParticiplesRegex.js` (see below). But for this there was a less obvious place to put this file.